lib/Backend/Commands.hs

```diff
@@ -436,6 +446,7 @@ tagCmd
   => Config -> TagOptions -> Sem r TagResult
 tagCmd config (TagOptions qEntryPath@(QualifiedPath backendNameMb entryPath) tag delete) = do
   backend <- getBackend config backendNameMb
+  validatePath backend entryPath
```

Comment:
Forgot to validate the path in `deleteCmd`

Reply:
done